Dockerfile

```diff
@@ -37,6 +37,8 @@ RUN set -eux; \
     \
     apk del --no-network .build-deps;
 
+COPY --from=ochinchina/supervisord:latest /usr/local/bin/supervisord /usr/bin/supervisord
```

Comment:
this `ochinchin/supervisord` dependency dependency looks to be problematic. doesn't to have a build for the architecture arm64 and does not updated for more than 4 years